geotrek/trekking/tests/data/file_good_encoding.kml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:gx="http://www.google.com/kml/ext/2.2" xmlns:kml="http://www.opengis.net/kml/2.2" xmlns:atom="http://www.w3.org/2005/Atom">
+<Document>
+<description>
+	Testé dans le cadre des parseurs donc les caractères: & à @ é ê è doivent être bien affichés
```

Comment:
Characters on this line are not displayed properly. Is the file encoded with UTF-8?

Reply:
Odd, this snippet displays characters well but they are messed up in the "Files changed" tab...

Reply:
AH oui la je sais pas, mais le test de ce fichier porte sur un encodage en utf-8 ...
Donc ça devrait être bon